### ISSUE TITLE
Code Reproduce Issues

### ISSUE THREAD

**Issue by EEE1even** [dll-wu/IS-CSE#4]
I tried to reproduce the code, and while debugging I realized that there was an if statement in the model that I could never get to, but the code in that if statement involved a key implementation step of the paper. I would like to know where I am going wrong .

the code part in `IS-CSE/simcse/model.py`,line 257:

```
    batch_size, hidden_size = z1.shape
    cls.lbuf = torch.cat([cls.lbuf[batch_size:,:], z2_normalized.detach()], dim=0)
    if cls.lbuf[0].norm() > 0:
        z2_star = find_neighbors_and_aggregation(cls, z2_normalized, mode='knn')
        cos_sim_lgroup = cls.sim(z1.unsqueeze(1), z2_star.unsqueeze(0))
        smoth_loss_labels = torch.arange(cos_sim_lgroup.size(0)).long().to(cls.device)
        loss_fct_lg = nn.CrossEntropyLoss()
        smoth_loss = loss_fct_lg(cos_sim_lgroup, smoth_loss_labels)
```

when i debug , the value `cls.lbuf[0].norm()` always equal 0 .This results in the if statement never being executed and the smoothed embedding and loss described in the paper is not obtained

I'd appreciate it if you could answer my issues ！
